lib/Tool/Console.php

```diff
@@ -410,6 +414,45 @@ protected static function execInBackgroundUnix($cmd, $outputFile)
         return $pid;
     }
 
+    /**
+     * @static
+     *
+     * @param string $cmd
+     * @param string $outputFile
+     *
+     * @return int
+     */
+    protected static function execInBackgroundDarwin($cmd, $outputFile)
```

Comment:
This function is actually a 1:1 copy of `execInBackgroundUnix` except the `nohup` command. I guess it would be better to just add an additional parameter `$useNohup` to the existing method.

Reply:
Has been fixed